src/components/bookdetails.js

```diff
@@ -1,10 +1,15 @@
 import React from 'react';
 import { useDispatch } from 'react-redux';
 import PropTypes from 'prop-types';
-import { removeBook } from '../redux/books/booksSlice';
+import { removeBook, deleteBooks } from '../redux/books/booksSlice';
 
 function BookDetails({ book }) {
   const dispatch = useDispatch();
+
+  const handleDelete = (endPoint) => {
+    dispatch(deleteBooks(endPoint))
+      .then(dispatch(removeBook(endPoint)));
+  };
```

Comment:
- [ ] [REQUIRED] Good job so far! Kindly when I try to delete elements they `disappear` but they are not totally `removed` from `API` because after updating the web page and switching the nav elements they are restored. Please fix it 🙏🏼

![ezgif-1-b5a2f576b8](https://github.com/Mikey1-nuel/Bookstore/assets/63915024/4c29ca44-b9fc-4b97-a1f3-dcd44e486e79)